### PR TITLE
Field table rewrite

### PR DESCRIPTION
**Pull request by GFDL-Eric**
field_to_yaml.py can now be called directly from command line. I've also edited the shebang as per PR #2

### CODE REVIEW

File: field_table/field_to_yaml.py [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-#field_table_suffix = '_norayleigh'
-field_table_suffix = '_esm4'
 # Necessary to dump OrderedDict to yaml format
 yaml.add_representer(OrderedDict, lambda dumper, data: dumper.represent_mapping('tag:yaml.org,2002:map', data.items()))
 
+if len(sys.argv) > 1:
+  field_table_name = sys.argv[1]
+else:
+  field_table_name = 'field_table'
+print(field_table_name)
```
Comment (ceblanton) [inline field_table/field_to_yaml.py]: Could this `print` be removed or hidden with the verbose flag?

Comment (ceblanton) [inline field_table/field_to_yaml.py:14]: Could this unneeded `print` be removed?

**Review by ceblanton — CHANGES_REQUESTED** [review]
Ran fine on a sample field_table.  Suggest removing the `print` of the filename for consistency with the other tools.

File: field_table/field_to_yaml.py (hunk 1 above)
Comment (GFDL-Eric) [reply field_table/field_to_yaml.py]: done

**Review by ceblanton — APPROVED** [review]

**Pull request merged**